open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/scheduler/AssetLineageUpdateJob.java

```diff
@@ -63,7 +64,7 @@ private void performTask(LocalDateTime localDateTime, JobDataMap dataMap) throws
             Optional<Long> storedAssetLineageUpdateTime = lineageGraph.getAssetLineageUpdateTime();
             Optional<LocalDateTime> assetLineageLastUpdateTime = getAssetLineageLastUpdateTime(configAssetLineageDefaultTime,
                     storedAssetLineageUpdateTime, localDateTime);
-            log.debug(RUN_ASSET_LINEAGE_UPDATE_JOB, assetLineageLastUpdateTime.get(), ZoneId.systemDefault().getId());
```

Comment:
Interesting I see you appear to be storing local time and then putting out the time zone. Egeria stores everything in UTC, this simplifies everything - you could ,of course, still display it in local time. It looks like you are using the time zone of the lineage server not the time zone of the asset on the system it lives on. UTC avoids any potential confusion.

Reply:
> @lpalashevski Interesting I see you appear to be storing local time and then putting out the time zone. Egeria stores everything in UTC, this simplifies everything - you could ,of course, still display it in local time. It looks like you are using the time zone of the lineage server not the time zone of the asset on the system it lives on. UTC avoids any potential confusion.

Not quite, we are storing the unix epoch millis which can be considered UTC from logical perspective. 

The convertion to local zonned time instant before it gets to the api call. Maybe we need to revisit the asset-lineage api and change the interface to accept absolute time instead of zonned time (then we can avoid converting back and forth). I am not sure but I think originally the idea was to have the endpoint accept local system times for better usability. We also have default updateTime parameter that can be provided by system admin when OLS is configured and this is the only place where we might need user-readable time.

Now thinking again the action performed by calling publish endpoint will be not ivoked directly by user but instead other tools (component) - then changing the parameter to absolute time (unix epoch millis) makes more sense.